sdk/resourcemanager/azure-resourcemanager-samples/src/test/java/com/azure/resourcemanager/samples/ComputeSampleTests.java

```diff
@@ -207,6 +207,7 @@ public void testCloneVirtualMachineToNewRegion() {
     }
 
     @Test
+    @Disabled("Creation of managed images are not supported for virtual machine with TrustedLaunch security type.")
```

Comment:
same as https://github.com/Azure/azure-sdk-for-java/pull/42308/files#r1800474602